default/de/publication_520.md

```diff
@@ -10,5 +10,5 @@ Enter the beginning date and (if applicable) the ending date of an item. If the
 
 ##### Examples
 
-- Vol. 1, no. 1 (Apr. 1981)-
+- Vol. Vol. 1, no. 1 (Apr. 1981)-
```

Comment:
@lpugin There's something weird going on here -- a lot of things are duplicated (see the "Ph.D." above too... )

Reply:
Hmm... I haven't change anything in there. At least I did not mean to....

Reply:
It seems like there's something messed up in Crowdin?

Reply:
<img width="717" alt="image" src="https://github.com/user-attachments/assets/967fbe32-88c1-4947-970f-3b2b6534c652">